site/app/templates/officeHoursQueue/QueueFooter.twig

```diff
@@ -311,5 +313,19 @@
       });
   }
 
+  function updateMessage(queue_code, alert, message, course) {
+    {% if viewer.inQueue() %}
+        let current_code = '{{ viewer.getCurrentQueueCode() }}'
+        if (current_code === queue_code) {
+            if (alert) {
+                queueMessage(message, course);
+            }
+            updateQueueStatus();
```

Comment:
A solution I see to this problem is to create an update queue message function. You can create an endpoint returning json info of the latest queue message. Then the function will call that endpoint and know the message which can be passed to the queue message function. I don't necessarily think course is necessary. If you click on the notification I believe it takes you directly to that page.